theme/static/js/resourceLandingAjax.js

```diff
@@ -417,10 +417,9 @@ function remove_aggregation_ajax_submit(url) {
 }
 
 function getResourceMetadata() {
-    const res_id = $("#short-id").val();
     return $.ajax({
         type: "POST",
-        url: '/hsapi/_internal/' + res_id + '/get-metadata/',
+        url: '/hsapi/_internal/' + SHORT_ID + '/get-metadata/',
```

Comment:
is this a CONSTANT or variable?

Reply:
This is a constant which is meant to replace all occurrences of res_id. I'll make sure to update these occurrences.